builder/vmware/common/driver_config_test.go

```diff
@@ -74,8 +73,14 @@ func TestDriverConfigPrepare(t *testing.T) {
 	for _, c := range tc {
 		t.Run(c.name, func(t *testing.T) {
 			errs := c.config.Prepare(interpolate.NewContext())
-			if !reflect.DeepEqual(errs, c.errs) {
+			if len(errs) != len(c.errs) {
 				t.Fatalf("bad: \n expected '%v' \nactual '%v'", c.errs, errs)
+			} else {
```

Comment:
We probably don't need the `else` here since `t.Fatalf` ends the test immediately

Reply:
Modified in d544221 to align both files.